_content/av/art-of-thailand_mcgill-forrest.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Arts of Thailand"
+authors:
+  - "Forrest McGill"
+drive_links:
+  - "https://drive.google.com/file/d/1oMOqxK6zrvjkanSX4mFgd5sgvmAm97Nh/view?usp=drive_link"
```

Comment:
For the external_url I've uploaded the video to archive org: https://archive.org/details/arts-of-thailand-2005-02-04-forrest-mcgill